### ISSUE TITLE
JavaFX

### ISSUE THREAD

**Issue by linuxboss182** [linuxboss182/SoftEng-2017#4]
Somebody should create a basic JavaFX frame. 

For now, the frame should just contain a button that calls our print team members function.

Whoever takes this issue, please make this feature branch so people can collaborate on it and do not merge into master branch until the feature is fully functional.

**Comment by linuxboss182** [comment]
The  Readings and Tasks - Requirements.docx  for Wednesday contains a tutorial on how to do this. 

Copying that here;

Scene Builder Installation – IntelliJ (JavaFX is already installed with IntelliJ)
Install Scene Builder from http://gluonhq.com/open-source/scene-builder/  
Opening Scenebuilder with IntelliJ
http://docs.oracle.com/javafx/scenebuilder/1/use_java_ides/sb-with-intellij.htm

Do the Scene Builder tutorial (up to 5.4 Preview of your Design)
https://examples.javacodegeeks.com/desktop-java/javafx/scene/javafx-scene-builder-tutorial/

Read a brief overview of Scene Builder containers http://docs.oracle.com/javafx/scenebuilder/1/user_guide/library-panel.htm 

Oracle Scene Builder reference web page:
http://www.oracle.com/technetwork/java/javase/downloads/javafxscenebuilder-info-2157684.html
